docs/general/builtins/scalar.rst

```diff
@@ -833,9 +850,9 @@ are given.
 
 .. NOTE::
 
-   The ``CURRENT_TIMESTAMP`` will be evaluated  using javas
-   ``System.currentTimeMillis()``. So its actual result depends on the
-   underlying operating system.
+   The ``CURRENT_TIMESTAMP`` and ``CURRENT_TIME`` expressions will be evaluated
+   using JMV's ``Clock.systemUTC().instant()``. So its actual result depends on
+   the underlying operating system, hardware and the JVM implementation.
```

Comment:
Maybe we could omit the implementation details here and just state that the time returned depends on the system clock and JVM implementation.